examples/nrf/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
```

Comment:
accidentally added file

Reply:
Ack, sorry. That was a merge error. Fixed.